src/game_functions/board_functions.c

```diff
@@ -0,0 +1,90 @@
+#include "board_functions.h"
+#include <stdbool.h>
+#include <stdlib.h>
+
+
+struct positionFromIndex calcPositionFromIndex (int index, int width) {
```

Comment:
With a lot of these functions, although the javadocs comments for the functions specify what each ``width`` and ``height`` are, it is unclear initially what these refer to.  Perhaps adding more descriptive variable names such as ``board_width`` or ``board_height`` will alleviate this issue.

Reply:
I chose to keep it generic since it's a utility function. That function applies to any grid, not just the board. Let me know if you still think changing the name is good.

Reply:
I think that is fine